src/main/java/com/epam/rd/autotasks/springemployeecatalog/extractor/EmployeeResultSetExtractorManagersManager.java

```diff
@@ -0,0 +1,91 @@
+package com.epam.rd.autotasks.springemployeecatalog.extractor;
+
+import com.epam.rd.autotasks.springemployeecatalog.domain.Department;
+import com.epam.rd.autotasks.springemployeecatalog.domain.Employee;
+import com.epam.rd.autotasks.springemployeecatalog.domain.FullName;
+import com.epam.rd.autotasks.springemployeecatalog.domain.Position;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.ResultSetExtractor;
+import org.springframework.stereotype.Component;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.DEPARTMENT;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.FIRST_NAME;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.HIRE_DATE;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.ID;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.LAST_NAME;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.LOCATION;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.MANAGER;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.MIDDLE_NAME;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.NAME;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.POSITION;
+import static com.epam.rd.autotasks.springemployeecatalog.constants.Constant.SALARY;
+import static java.sql.Types.NULL;
+
+@Component
+public class EmployeeResultSetExtractorManagersManager implements ResultSetExtractor<List<Employee>> {
```

Comment:
Class is rather similar with  EmployeeResultSetExtractor. With similar methods and logic. Should be refactored